setup.R

```diff
@@ -38,3 +38,5 @@ bttn_remove <-  list('select2d', 'lasso2d', 'zoomIn2d', 'zoomOut2d',
 
 # Read in data for waiting_list_chart
 Join_all_long <- read_csv("data/Join_all_long.csv",show_col_types = FALSE)
+# Read in data for waiting_list_chart
+Join_all_long2 <- read_csv("data/Join_all_long2.csv",show_col_types = FALSE)
```

Comment:
Do you still need both of these datasets? Or could you remove one? 

It's probably also worth giving the data a clearer name as when you add more data / charts / tabs, it won't be obvious that `join_all_long` refers to waiting list data. (Unless this dataset isn't just waiting list of course!)